cms/djangoapps/contentstore/views/preview.py

```diff
@@ -285,6 +285,7 @@ def _studio_wrap_xblock(xblock, view, frag, context, display_name_only=False):
             'can_edit': context.get('can_edit', True),
             'can_edit_visibility': context.get('can_edit_visibility', True),
             'can_add': context.get('can_add', True),
+            'can_move': context.get('is_unit_page', False),  # move action is not allowed other than unit page
```

Comment:
When a page is unit page, it has `True` set to `is_unit_page` field of the `xblock_info`. So `context.get('is_unit_page', False)` would either give `True` (if unit page) or `False`